### PR TITLE
EOI form

### PR DESCRIPTION
**Pull request by oliverjam**
A working form that submits to Airtable. Also the form handler now supports custom email templates based on the name of the form that submitted it

- [x] reject ineligible people and redirect them to a custom error page.

### CODE REVIEW

**Comment by sofer** [comment]
The sticky yellow "Express interest" banner is still linking to the old google form. Do we want to update that?

(for my own reference: /forms/express-interest)

**Comment by oliverjam** [comment]
@sofer no this isn't ready to be used yet. Just preliminary work to get things ready (see Discord for more context)

File: backend/forms/email-templates/express-interest.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module.exports = ({ data, readableData }) => {
+  return {
+    subject: "Founders and Coders Expression of Interest",
+    text: `Hi ${data.name},
+
+Thank you for your interest in the Winter 2021 cohort of the Founders and Coders programme.
```
Comment (yvonne-liu) [inline backend/forms/email-templates/express-interest.js:6]: Should this say January 2022?

**Review by yvonne-liu — CHANGES_REQUESTED** [review]

File: backend/forms/email-templates/express-interest.js (hunk 1 above)
Comment (oliverjam) [reply backend/forms/email-templates/express-interest.js:6]: Damn I should've searched across all files 🤦

**Review by yvonne-liu — APPROVED** [review]

**Pull request merged**